src/Lesson_1/Lesson_1.java

```diff
@@ -0,0 +1,107 @@
+package Lesson_1;
+
+public class Lesson_1 {
+    public static void main(String[] args) {
+        variableUser ();
+
+        System.out.println("Результат " + ABCD());
+
+        boolean resultXY = XY(20, 20);
+        System.out.println("Результат (X+Y)>=10 && (X+Y)<= 20 " + resultXY);
+
+        int it = -10;
+        integerNumber(it);
+
+        boolean resItBool = intBooleanNumber(it);
+        System.out.println("Результат  " + resItBool);
+
+        HelloString("В.В. Путин");
+        int years = 2021;
+        leapYears (years);
+    }
+
+
+    // Создать переменные всех пройденных типов данных и инициализировать их значения.
+    private static void variableUser (){
+
+        char ch = 'A';
+        String ST = "Создать переменные всех пройденных типов данных и инициализировать их значения:";
+        int it = 1;
+        float fl = 3.14159265f;
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [CNT: Rough value of known constant found (CNT_ROUGH_CONSTANT_VALUE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#cnt-rough-value-of-known-constant-found-cnt-rough-constant-value).
It's recommended to use the predefined library constant for code clarity and better precision.